src/api/controllers/admin/admin.setAdmin.js

```diff
@@ -0,0 +1,55 @@
+const isAdmin = require('../../middlewares/isAdmin')
+const isAuth = require('../../middlewares/isAuth')
+const errorHandler = require('../../utils/errorHandler')
+
+/**
+ * PUT /admin/setAdmin/:id
+ *
+ * Response: none
+ * 
+ */
+module.exports = app => {
+  app.put('/admin/setAdmin/:id', [isAuth(), isAdmin()])
+
+  app.put('/admin/setAdmin/:id', async (req, res) => {
+    const { Permission } = req.app.locals.models
+
+    try {
+      if(req.body.admin === null) {
+        return res
+        .status(400)  // Bad request
+        .json({ error: 'BAD_REQUEST' })
+        .end()
+      }
+      else if(req.params.id === req.user.id) {
```

Comment:
```suggestion
     if(req.params.id === req.user.id) {
```
Le else est inutile : le if précédent finit par un "return"